specifyweb/frontend/js_src/lib/components/Merging/InvalidMergeRecords.tsx

```diff
@@ -0,0 +1,83 @@
+import { RA } from '../../utils/types';
+import { AnySchema, SerializedResource } from '../DataModel/helperTypes';
+import { H2, Ul } from '../Atoms';
+import { FormattedResource } from '../Molecules/FormattedResource';
+import { deserializeResource } from '../DataModel/helpers';
+import React from 'react';
+import { OverlayContext } from '../Router/Router';
+import { Dialog } from '../Molecules/Dialog';
+import { recordMergingTableSpec } from './definitions';
+import { dialogIcons } from '../Atoms/Icons';
+import { Button } from '../Atoms/Button';
+import { commonText } from '../../localization/common';
+import { mergingText } from '../../localization/merging';
+import { Tables } from '../DataModel/types';
+import { TableIcon } from '../Molecules/TableIcon';
+
+function InvalidMergeRecords({
+  resources,
+  tableName,
+  body = '',
+}: {
+  readonly tableName: keyof Tables;
+  readonly resources: RA<SerializedResource<AnySchema>>;
+  readonly body?: string;
+}): JSX.Element {
+  return (
+    <div className="flex flex-1 flex-col gap-4">
+      <H2>{body}</H2>
```

Comment:
the dialog renders an `<h2>`, thus this should be an `<h3>` or a `<p>`

Reply:
this was marked as resolved but wasn't fixed.

if the only reason you are using H2 is styling, then you can get the same styling for p or h3 relatively easily:
```tsx
<h3 className={className.headerPrimary}>{body}</h3>
```

(just copied from the definition of H2:)
https://github.com/specify/specify7/blob/207df9a091c59cd04e4e4b1c3eb111c75c74f68a/specifyweb/frontend/js_src/lib/components/Atoms/index.tsx#L66C25-L67